fe/src/main/java/org/apache/doris/catalog/FsBroker.java

```diff
@@ -0,0 +1,129 @@
+package org.apache.doris.catalog;
+
+import org.apache.doris.common.io.Text;
+import org.apache.doris.common.io.Writable;
+import org.apache.doris.system.BrokerHbResponse;
+import org.apache.doris.system.HeartbeatResponse.HbStatus;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+//Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
license header should be before package and import

Reply:
Done